packages/babel-preset-env/src/index.js

```diff
@@ -49,6 +49,8 @@ const pluginListWithoutProposals = filterStageFromList(
   proposalPlugins,
 );
 
+const validEcmascriptTargets = new Set(["5", "2015", "2016", "2017", "2018"]);
```

Comment:
Also, we can have a single source of truth and consider this Set for build-data script. Or vice versa. It could reduce steps needed to add a new version in the future.

Reply:
Good idea

Reply:
After investigating the implementation of this, I worry about this being overly abstracted, especially since the data source isn't that consistent and heuristics can be a bit sub-optimal. If the format of the compat-tables change, you might be left with a bad abstraction, so I'm not sure sacrificing clarity/simplicity for DRY would be the right call.